src/util/irep_ids.def

```diff
@@ -212,6 +212,7 @@ IREP_ID_ONE(NULL)
 IREP_ID_ONE(null)
 IREP_ID_ONE(nullptr)
 IREP_ID_ONE(c_enum)
+IREP_ID_ONE(underlying_type)
```

Comment:
Should this be something like `enum_underlying_type` or is there a reason is should not be specific to enums?

Reply:
No particular reason, I've changed it to `enum_underlying_type`.